### ISSUE TITLE
451: DEPLOYMENT_DISABLED

### ISSUE THREAD

**Issue by MelsonMusk** [davidivkovic/tailwindui#4]
Hi David,

The site seems to be down for a while with the following error:

This Deployment has been disabled.

<img width="524" alt="Screenshot 2023-02-03 at 11 07 54 PM" src="https://user-images.githubusercontent.com/9590632/216638348-dacb647b-c7fa-435b-9468-0a3ee02da89a.png">

Any plan for a fix?

Thanks

**Comment by davidivkovic** [comment]
Hey Melon, thank you for alerting me!

Unfortunately I am seeing a 5x increase in incoming requests today, compared to average traffic.
This probably triggered a bandwidth quota limit as this site is on the free Vercel plan.

The deployment should be online now, and I will be monitoring to find out what has happened.

**Comment by MelsonMusk** [comment]
Thanks a lot! It's working now.

I see that Tailwind has an update in January.

Could you please also make an update if you have time?

Thanks in advance!

**Comment by davidivkovic** [comment]
I haven't found it online yet, didn't actually purchase this myself. If I find it I will upload for sure. Someone asked me if this is legal to use, you can do whatever you like though, it's just a collection of some classes.

**Comment by MelsonMusk** [comment]
Sure, thanks!

**Comment by charlescarrasco** [comment]
If you can, maybe look into hosting this statically on Cloudflare Pages. I think it's disabled again.

**Comment by davidivkovic** [comment]
> If you can, maybe look into hosting this statically on Cloudflare Pages. I think it's disabled again.

Yeah I am currently aware of that, I redeployed it again. Technically this Vercel deployment is also fully static, I am not sure what is happening. Might try Netlify also.

**Comment by davidivkovic** [comment]
Live on https://tailwindui.surge.sh

**Comment by charlescarrasco** [comment]
LOL, this is becoming a game of cat and mouse now, try GitHub Pages or Cloudflare Pages?

**Comment by davidivkovic** [comment]
@charlescarrasco 
Any western company will take it down. Know any chinese or russian static site hosting?

**Comment by charlescarrasco** [comment]
I doubt you can find any Chinese ones @davidivkovic but it could be possible to find Russian or Indian ones, best bet is one of those sketchy cPanel hosting sites like https://www.infinityfree.net, never used them but there's a bunch like this. I'd recommend you use an email gen when creating any accounts with them I wouldn't give them my real info.

**Comment by hotcheetos1337** [comment]
https://tailwind-ui-roan.vercel.app/

Includes the latest marketing components! There are also various fixes for issues such as pages being opened in a modal and snippet language selection not being saved.

**Comment by davidivkovic** [comment]
If you find something from China please share here, for now all three websites got taken down.
